src/Airport.js

```diff
@@ -0,0 +1,49 @@
+const Plane = require('./Plane');
+
```

Comment:
The Airport class is tightly coupled with the Plane class, so on review I would try to adjust this